middleware/jwt.go

```diff
@@ -225,3 +230,17 @@ func jwtFromCookie(name string) jwtExtractor {
 		return cookie.Value, nil
 	}
 }
+
+// jwtFromBody returns a `jwtExtractor` that extracts token from the request body.
+func jwtFromBody(name string) jwtExtractor {
+	return func(c echo.Context) (string, error) {
+		reqBody, _ := ioutil.ReadAll(c.Request().Body)
+		jsonBody := map[string]string{}
+		err := json.Unmarshal(reqBody, &jsonBody)
+		if err != nil || len(jsonBody[name]) == 0 {
```

Comment:
I think these are different things.
- in one case, we had error of unmarshaling
- in another the absence of a JWT token

Reply:
@im-kulikov you're correct! Should I add new errors for reading body and unmarshaling